atm.js

```diff
@@ -1,21 +1,57 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+//function gaATM with OOP
+function gaATM() {
+  this.balance = 0;
+  this.checking = {
+    input: $("#newChecking"),
```

Comment:
the console is giving me an error: `atm.js:5 Uncaught ReferenceError: $ is not defined`

This means it doesn't understand what $ is referring it, it isn't defined. It's not picking up the jQuery library. I think your missing a script tag for jQuery cdn